BUILD.md

```diff
@@ -0,0 +1,16 @@
+# Build
+
+## Dependencies
+
+- [jEnv](http://www.jenv.be/)
```

Comment:
Haven't used this tool before, do we really want to require an additional dependency for running tests?